recipes/pytest-pyvista/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pytest-pyvista" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pytest_pyvista/pytest_pyvista-{{ version }}.tar.gz
+  sha256: 43e6f0a85b6ff23aff51b7f750528de0fdda6da2011a94b636f05d8498fe38a0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

Comment:
@tkoyama010 Aren't you instead using `flit` to build and install the package?

Reply:
@ocefpaf Go some spare cycles for some sage advice?

I've never seen `flit` being used on conda-forge before to build a package, is there support to do this, and if so what pattern do we adopt here? Thanks, as always 😄

Reply:
No problem, found a tonne of examples, see [here](https://github.com/search?q=org%3Aconda-forge+%2Fflit-core%2F+path%3A%2F%5Erecipe%5C%2F%2F&type=code)

Reply:
Yup, so the `script` call is fine with `pip`, so don't change that 👍

Reply:
Thanks for your thought!